core/templates/pages/exploration-editor-page/modal-templates/save-version-mismatch-modal.controller.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright 2020 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Controller for version mismatch modal.
+ */
+
+angular.module('oppia').controller('SaveVersionMismatchModalController', [
+  '$log', '$scope', '$timeout', 'ExplorationDataService',
+  'LostChangeObjectFactory', 'WindowRef', 'lostChanges',
+  function(
+      $log, $scope, $timeout, ExplorationDataService,
+      LostChangeObjectFactory, WindowRef, lostChanges) {
+    var _refreshPage = function(delay) {
+      $timeout(function() {
+        WindowRef.nativeWindow.location.reload();
+      }, delay);
+    };
+    // When the user clicks on discard changes button, signal backend
+    // to discard the draft and reload the page thereafter.
+    $scope.discardChanges = function() {
+      ExplorationDataService.discardDraft(function() {
+        _refreshPage(20);
```

Comment:
Avoid magic numbers, move 20 to a constant.

Reply:
Done.